qmlbind/include/qmlbind/application.h

```diff
@@ -65,6 +65,35 @@ QMLBIND_API void qmlbind_application_release(qmlbind_application *self);
 QMLBIND_API int qmlbind_application_exec(qmlbind_application *self);
 
 /*!
+ * \brief Sets the application's name.
```

Comment:
You could add something this to the documentation, like you wrote in the pull request message:

> This primarily allows for proper file names for saving and retrieving settings with the Settings QML Type